src/stores/attributes-customer-address-templates.md

```diff
@@ -5,46 +5,36 @@ title: Customer Address Templates
 
 You can modify the template that controls the format of customer billing and shipping addresses that appear on printed invoices, shipments, and refunds, as well as in the addresses book of the customer account. If you need to include additional information, you can create [custom attributes]({{ site.baseurl }}{% link stores/attributes-customer.md %}) that are associated with the customer account and [address]({{ site.baseurl }}{% link stores/attributes-customer-address.md %}), and incorporate them into the template.
 
-## Example 1: Short Format
+## Example 1: short format
 
 For Text One Line Template:
 
-        `{% raw %}{{depend prefix}}{{var prefix}} {{/depend}}{{var firstname}} {{depend middlename}}{{var middlename}} {{/depend}}{{var lastname}}{{depend suffix}} {{var suffix}}{{/depend}}, {{var street}}, {{var city}}, {{var region}} {{var postcode}}, {{var country}}{% endraw %}`
+        {% raw %}{{depend prefix}}{{var prefix}} {{/depend}}{{var firstname}} {{depend middlename}}{{var middlename}} {{/depend}}{{var lastname}}{{depend suffix}} {{var suffix}}{{/depend}}, {{var street}}, {{var city}}, {{var region}} {{var postcode}}, {{var country}}{% endraw %}
```

Comment:
Formatting is off here. We should not be displaying the raw tags. I suspect the template needs to be in a code block and the raw tags outside of that.

Reply:
The `{% raw %}` is a liquid tag and isn't displayed even within the ` enclosure. However, I changed this to use the fenced in block and the raw designation outside of that. The display is exactly the same. The linting rules don't like it, but that happens with some of our other liquid tags too.